sync2/rangesync/dumbset.go

```diff
@@ -298,14 +301,13 @@ func (ds *DumbSet) SplitRange(x, y KeyBytes, count int) (SplitInfo, error) {
 	}, nil
 }
 
-// Empty implements OrderedSet.
-func (ds *DumbSet) Empty() (bool, error) {
-	return len(ds.keys) == 0, nil
-}
-
-// Items implements OrderedSet.
-func (ds *DumbSet) Items() SeqResult {
-	return MakeSeqResult(ds.keys)
+// SetInfo implements OrderedSet.
+func (ds *DumbSet) SetInfo() (RangeInfo, error) {
+	ri, _, err := ds.getRangeInfo(nil, nil, -1)
+	if err != nil {
+		panic("unexpected error in SetInfo: " + err.Error())
+	}
```

Comment:
Why panic instead of returning the error?

Reply:
Fixed